### PR TITLE
Rearranging Files

### PR DESCRIPTION
**Pull request by YaronFruchtmann**
Required items, please complete
   
   Change(s):
   - Process Event
   - Registry Event
   - Some fields label changed Enum*a*rated to Enum*e*rated

### CODE REVIEW

File: ASIM/dev/ASimTester/ASimTester.csv [hunk 1]
```diff
@@ -516,3 +516,145 @@ TargetUrl,string,Optional,FileEvent,,,
...
+DvcId,string,Optional,RegistryEvent,,,
+DvcIpAddr,string,Recommended,RegistryEvent,,,
+DvcMacAddr,string,Optional,RegistryEvent,,,
+DvcOs,string,Optional,RegistryEvent,,,
+DvcOsVersion,string,Optional,RegistryEvent,,,
+EventCount,int,Mandatory,RegistryEvent,,,
+EventEndTime,datetime,Alias,RegistryEvent,,,TimeGenerated
+EventOriginalType,string,Optional,RegistryEvent,,,
+EventOriginalUid,string,Optional,RegistryEvent,,,
+EventProduct,string,Mandatory,RegistryEvent,,,
```
Comment (oshezaf) [inline ASIM/dev/ASimTester/ASimTester.csv:633]: For event product I do maintain a list as enumerated.

File: ASIM/dev/ASimTester/ASimTester.csv [hunk 2]
```diff
@@ -516,3 +516,145 @@ TargetUrl,string,Optional,FileEvent,,,
...
+TargetUserSessionId,string,Optional,ProcessEvent,string,,
+TimeGenerated,datetime,Mandatory,RegistryEvent,,,
+_ResourceId,string,Mandatory,RegistryEvent,,,
+Type,string,Mandatory,RegistryEvent,,,
+EventMessage,string,Optional,RegistryEvent,,,
+AdditionalFields,dynamic,Optional,RegistryEvent,,,
+Dvc,string,Mandatory,RegistryEvent,,,
+DvcHostname,string,Recommended,RegistryEvent,,,
+DvcId,string,Optional,RegistryEvent,,,
+DvcIpAddr,string,Recommended,RegistryEvent,,,
```
Comment (oshezaf) [inline ASIM/dev/ASimTester/ASimTester.csv:625]: You should the relevant Logical types (IP address, Hostname etc.) see the schemas I created or the data tester script.

**Review by oshezaf — CHANGES_REQUESTED** [review]

**Comment by vmanojreddy** [comment]
@YaronFruchtmann Please resolve conflicts. thanks!!!

**Comment by vmanojreddy** [comment]
@YaronFruchtmann Please resolve conflicts. thanks!!!

**Pull request closed without merging**